j2/rules/cap_customer_prospect.yaml

```diff
@@ -163,6 +303,9 @@ updatereq:
   - label: Has_Ingestion_Event
     fromVertexLabel: Person.Natural
     toVertexLabel: Event.Ingestion
+  - label: Has_Ingestion_Event
```

Comment:
Does this work, or do we need one for each type of consent event as below?

Reply:
sure ... forgot to modify that ... I would say ... we need one for each Event.Consent ... right?